rootfs/etc/cont-init.d/15-config-postfix.sh

```diff
@@ -257,6 +257,20 @@ EOL
 chmod o= /etc/postfix/mysql-virtual-alias-domains-and-subdomains.cf
 chgrp postfix /etc/postfix/mysql-virtual-alias-domains-and-subdomains.cf
 
+if [ "$POSTFIX_STRIP_SENDER_HEADER" = "true" ]; then
```

Comment:
Can you also add the default value for this env at https://github.com/anonaddy/docker/blob/d5210af9745d50494137d51109e98b8c4e6c566f/rootfs/etc/cont-init.d/00-env#L84-L92